generator/metaline_builder.cpp

```diff
@@ -0,0 +1,217 @@
+#include "generator/metaline_builder.hpp"
+#include "generator/routing_helpers.hpp"
+
+#include "indexer/classificator.hpp"
+
+#include "coding/file_container.hpp"
+#include "coding/varint.hpp"
+#include "coding/write_to_sink.hpp"
+
+#include "base/checked_cast.hpp"
+#include "base/logging.hpp"
+#include "base/string_utils.hpp"
+
+#include "defines.hpp"
+
+#include <cstdint>
+
+namespace
+{
+using Ways = std::vector<std::int32_t>;
```

Comment:
[Use <cstdint> and assume your implementation will put the symbols in the global namespace (very bad because not guaranteed).](https://stackoverflow.com/a/13643019/1297601)

What cstdint does, is it defines `std:*_t` namespaced types. Most of the time they are copies of `*_t` types provided by C++. If we don't need `std::` prefixes, then why do we keep including `<cstdint>`?